frontend/src/App.jsx

```diff
@@ -1,19 +1,16 @@
 import React from 'react';
-import { View } from 'react-native';
 import { registerRootComponent } from 'expo';
 import { Provider } from 'react-redux';
+import { createAppContainer } from 'react-navigation';
 import store from './store';
-import Troop from './Troop';
-import Menu from './Menu';
-import commonStyles from './common/styles';
+import TabNavigator from './Navigation';
+
+const AppContainer = createAppContainer(TabNavigator);
 
 function App() {
   return (
     <Provider store={store}>
-      <View style={commonStyles.container}>
-        <Menu />
-        <Troop />
-      </View>
+      <AppContainer />
```

Comment:
![Screen Shot 2019-12-11 at 11 56 40](https://user-images.githubusercontent.com/30794349/70590386-80d7cb80-1c0d-11ea-80dd-18095ccef3a3.png)

Active tab button looks like this. Please research if you can support safeArea, check if it's possible to highlight the active nav button in the safeArea as well.

Reply:
It seems like I cant apply any styles to the safeArea, but I can disable the bottom-safeArea for all kinds of devices. Should i do that?

Reply:
no, don't do that

Reply:
it's OK this way then.